packages/onboarding/src/step-container/index.tsx

```diff
@@ -174,6 +180,21 @@ const StepContainer: React.FC< Props > = ( {
 		'is-extra-wide-layout': isExtraWideLayout,
 	} );
 
+	let headers = null;
+	if ( formattedHeader ) {
+		headers = formattedHeader;
+	} else if ( headerText ) {
+		headers = (
+			<FormattedHeader
+				id="domains-header"
```

Comment:
Maybe its better to pass formattedHeader from the domains step

Reply:
Decided to go for it.